app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/User.scala

```diff
@@ -37,7 +37,8 @@ case class User(
   role: UserRole,
   createdAt: Timestamp,
   modifiedAt: Timestamp,
-  dropboxCredential: Option[String]
+  dropboxCredential: Option[String],
+  planetCredential: Option[String]
```

Comment:
Two is fine but if we add anymore I think we should add a `Credentials` case class with a bunch of `Option[String]`s in it